appengine-managed-runtime/src/main/java/com/google/apphosting/vmruntime/VmApiProxyEnvironment.java

```diff
@@ -126,95 +132,94 @@
 
   // Control the maximum number of concurrent API calls.
   // https://developers.google.com/appengine/docs/python/backends/#Python_Billing_quotas_and_limits
-  
+
   static final int MAX_CONCURRENT_API_CALLS = 100;
-  
+
   static final int MAX_PENDING_API_CALLS = 1000;
 
   /**
    * Mapping from HTTP header keys to attribute keys.
    */
-  static enum AttributeMapping {
+  enum AttributeMapping {
     USER_ID(
-        "X-AppEngine-User-Id",
-        "com.google.appengine.api.users.UserService.user_id_key",
-        "", false),
+      "X-AppEngine-User-Id",
```

Comment:
Did you reconfigure the formatter (to 2 spaces indent, 4 spaces wrap)? It didn't seem to do it here.